src/ci/docker/host-x86_64/x86_64-gnu-tools/checktools.sh

```diff
@@ -58,8 +58,9 @@ case $HOST_TARGET in
     # Strangely, Linux targets do not work here. cargo always says
     # "error: cannot produce cdylib for ... as the target ... does not support these crate types".
     # Only run "pass" tests, which is quite a bit faster.
-    python3 "$X_PY" test --stage 2 src/tools/miri --target aarch64-apple-darwin --test-args pass
-    python3 "$X_PY" test --stage 2 src/tools/miri --target i686-pc-windows-gnu --test-args pass
+    #FIXME: Re-enable this once CI issues are fixed
```

Comment:
Is there an issue that tracks this? Would be good to reference it here so that it's easy to check when the FIXME can be resolved.

Reply:
Also, would have been nice to ping @rust-lang/miri here; this change could mean a breakage on those targets only gets detected by Miri CI, and it would have been very confusing to figure out what happened when we are sure that Rust CI already covers these targets.

Reply:
Ah, I found the issue: https://github.com/rust-lang/rust/issues/127883

Added it to the PR description, too.

Reply:
Unless I messed something up, we should still be testing everything that we were testing. It's just being tested in a separate job (`x86_64-msvc-ext2`).

I only intended this to be very temporary but it seems the issues is persisting longer than I'd hoped.

Reply:
Oh, now I see, this got moved to `src/ci/github-actions/jobs.yml`... which is not where our tools testing logic usually lives so I missed that.